### PR TITLE
 Add pytest-ignore-flaky

### PR DESCRIPTION
**Pull request by rmcgibbo**

### CODE REVIEW

File: recipes/pytest-ignore-flaky/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  script: pip install .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest
+  run:
+    - python
+    - pytest
```
Comment (synapticarbors) [inline recipes/pytest-ignore-flaky/meta.yaml]: It looks like this package wants `pytest >=2.7`

File: recipes/pytest-ignore-flaky/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  noarch: python
+  number: 0
+  script: pip install .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest
```
Comment (synapticarbors) [inline recipes/pytest-ignore-flaky/meta.yaml]: I don't think this needs `pytest` as a `build` requirement.

File: recipes/pytest-ignore-flaky/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install .
```
Comment (synapticarbors) [inline recipes/pytest-ignore-flaky/meta.yaml]: I think if you're going to use `pip` here it should be:

```
script: pip install --no-deps .
```

or possibly:

```
python -m pip install --no-deps --ignore-installed .
```

see:
https://github.com/conda-forge/staged-recipes/issues/528

**Comment by rmcgibbo** [comment]
Thanks @synapticarbors.

**Comment by synapticarbors** [comment]
This looks ready to merge, but for some reason it's not triggering CircleCI, which we need to see in order to finalize.

**Comment by rmcgibbo** [comment]
Is there anything I can do? Retrigger?

**Comment by synapticarbors** [comment]
Sometimes there is something weird with certain personal accounts and how CircleCI is configured. I just made a copy of your PR in https://github.com/conda-forge/staged-recipes/pull/4516 and it looks like it's running ok, so we can just merge from there.

**Comment by rmcgibbo** [comment]
Thanks.

**Pull request closed without merging**